src/Microsoft.Azure.WebJobs.Logging.NuGet/WebJobs.Logging.nuspec

```diff
@@ -15,7 +15,7 @@
     <tags>Microsoft Azure WebJobs Jobs Logging windowsazureofficial Web</tags>
     <dependencies>
       <dependency id="Newtonsoft.Json" version="9.0.1"/>
-      <dependency id="WindowsAzure.Storage" version="7.2.1"/>
+      <dependency id="WindowsAzure.Storage" version="8.0.1"/>
```

Comment:
I pushed too early -- I rolled back those as well as the BindingRedirects in the app.config files.